R/calculate_recommended_interventions.R

```diff
@@ -1,123 +1,170 @@
 #' calculate_recommended_interventions
 #'
-#' @description Returns the LAGO recommended interventions for the next stage.
-#' The recommended interventions that aim to satisfy the outcome goal and/or the
+#' @description Calculates the LAGO recommended interventions for the next stage.
+#' The recommended interventions that aim to satisfy an outcome goal and/or a
 #' power goal.
 #'
-#' @param df data.frame, Input data
-#' @param outcome_name character, Outcome name
-#' @param outcome_type character, Outcome type, either "continuous" or "binary"
-#' @param glm_family character, Family for the GLM model
-#' For example: "binomial"
-#' For v1.1, we are only considering binomial family, and quasibinomial(link="logit")
-#' @param link character, the link option when glm_family is set to quasibinomial.
-#' It has to be one of the following:
-#' logit (default), probit, cauchit, log and cloglog.
-#' @param include_intercept boolean, Whether the intercept should be included in the
-#' fitted model.
-#' @param interventions_list character vector, Names of the intervention components
-#' For example: c("component1", "component2")
-#' @param center_characteristic_list character vector, Names of the center characteristics
-#' For example: c("characteristic1")
-#' @param center_characteristic_list_for_optimization numeric vector, Given values of
-#' the center characteristics
+#'
+#' ## required arguments:
+#' @param data data.frame: input data frame.
+#' @param outcome_name character: column name of the outcome of interest.
```

Comment:
A question: Do we only support one outcome? Are there any code versions/ datasets that the LAGO team has done for multiple outcomes? If this is the case, should we specify that "Column name of **one** outcome of interest"?  I feel like in real life lots of users can get ambitious and want to do multiple at the same time so might as well specify it, that we deal with one at a time.

Reply:
Currently only one. I think we can support multiple outcome goals in later versions. PULESA actually has two outcome goals, and they are jointly optimized.